packages/sdk/src/swap/types.ts

```diff
@@ -90,7 +104,10 @@ export type SwapStatusResponse = CommonStatusFields &
         swapExecutedBlockIndex: string;
         egressAmount: string;
         egressScheduledAt: number;
-        egressCompletedAt: number;
-        egressCompletedBlockIndex: string;
+        egressScheduledBlockIndex: string;
+        broadcastRequestedAt: number;
+        broadcastRequestedBlockIndex: string;
+        broadcastSuccessAt: number;
+        broadcastSuccessBlockIndex: string;
```

Comment:
```suggestion
        broadcastSucceededAt: number;
        broadcastSucceededBlockIndex: string;
```

Reply:
i used `broadcastSuccess` because the event name is `BroadcastSuccess`. but dont have a strong opinion and will change